README.md

```diff
@@ -181,6 +182,10 @@ To run the unit tests:
 npm run test
 ```
 
+## Known Limitations
```

Comment:
As per UA guidelines, we should not use the word limitations. Rather we should change it to restrictions.